### ISSUE TITLE
Manual Install Guide

### ISSUE THREAD

**Issue by anonymoususer1001** [bbmaster123/10SM#13]
Not really an issue, but could you provide the location of all the files so that a manual install is at least possible? I ask because the release said a manual install is needed as "install.bat" only partially installs it. I really wanna use and test this so I can make the move to RTM builds as opposed to the initial release (22000.1).

**Comment by bbmaster123** [comment]
ah ok you must be looking at an older fork or something, you do not need the install.bat anymore, I have since removed it. Ended up caused more problems than I had time to fix. 

To install this, you will need to manually replace the files for now, no installer. I need some more free time so I can work on that. An installer is becoming a pretty requested feature so I will do it when I can.

For a manual install, each file is nested in its intended directory,  so for example if you download the whole repo as a zip, the jumpui.dll in the shellexperiences folder would go in C:\windows\shellexperiences 
and so on for all the files

hope that helps :)
